casaos-fix-docker-api-version/run.sh

```
@@ -1424,30 +1445,89 @@ show_usage() {
+    if [ -z "$SUDO" ]; then
+      return 0
+    fi
+  fi
+
+  echo "Configuring policy-rc.d to prevent service auto-start..."
+  
+  if [ -f /usr/sbin/policy-rc.d ]; then
+    # Back up existing policy-rc.d if it's not ours
+    if ! grep -q "Prevent docker and containerd" /usr/sbin/policy-rc.d 2>/dev/null; then
+      echo "Backing up existing policy-rc.d..."
+      $SUDO cp /usr/sbin/policy-rc.d /usr/sbin/policy-rc.d.backup.$(date +%s)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)